lib/matplotlib/_mathtext.py

```diff
@@ -2060,6 +2060,7 @@ def __init__(self):
         p.accent        <<= Group(
             Suppress(p.bslash)
             + oneOf([*self._accent_map, *self._wide_accents])
+            + Suppress(Optional(White()))
```

Comment:
If the code is not working without this explicit whitespace-skipping, that is a pyparsing bug.

Reply:
Ok. There _is_ a bug in pyparsing, but fixing it doesn't completely solve your problem.

What is the purpose of `leaveWhitespace()` here?

        p.symbol        <<= (p.single_symbol | p.symbol_name).leaveWhitespace()

It looks like this is the only element in `p.placeable` that suppresses whitespace-skipping. The new code detects this and instead of doing whitespace-skipping before the `MatchFirst`, it doesn't do whitespace-skipping, relying on all the separate elements to do it themselves (this is the performance optimization - if all the alternatives will want whitespace to be skipped, the `MatchFirst `will do it once, and then all the alternatives get a pass). Since `p.symbol` suppresses whitespace-skipping, and so makes `p.placeable` suppress whitespace-skipping, then when it is time to match `p.symbol`, the leading space has not been skipped, and so it fails.

So:
- I need to fix the bug in pyparsing - this will be in 3.0.4, which I will push out this evening.
- This is another pyparsing _improvement_ that needs to be documented in the "stuff that has changed for the better, but may affect your code" - I'll add that information in 3.0.4 also.
- Please revisit why you need to have `leaveWhitespace()` on `p.symbol`.

   If it is required, then I think you could remove the `Suppress(Optional(White()))` from `p.accent`, and instead change the `p.symbol` usage in `p.placeable` to:

        | Empty() + p.symbol  # Must be third to catch all named symbols and single

  (I tested this in my minimal repro example and that works.)
  But ideally, you could just remove the call to `leaveWhitespace()`.

Reply:
Agreed that removing leaveWhitespace seems to work; looking at the git history it seems to be a historical artefact of the times when symbol and symbol_name were not split, and was used for symbol_name.